apps/site-landing/src/containers/MainNavigation/index.tsx

```diff
@@ -1,15 +1,81 @@
 import React from 'react';
-import { GreaterThanMobileSegment, MobileSegment } from '@thecointech/shared/components/ResponsiveTool';
-import { MainNavigationMobile } from './MainNavigationMobile';
-import { MainNavigationGreaterThanMobile } from './MainNavigationGreaterThanMobile';
-
-export const MainNavigation = () =>
-  <>
-    <GreaterThanMobileSegment>
-      <MainNavigationGreaterThanMobile />
-    </GreaterThanMobileSegment>
-
-    <MobileSegment>
-      <MainNavigationMobile />
-    </MobileSegment>
-  </>
+import { Dropdown, Menu } from 'semantic-ui-react';
+import { defineMessage, defineMessages, FormattedMessage } from 'react-intl';
+import { Link, NavLink } from 'react-router-dom';
+import { HeaderLink } from '@thecointech/site-base/components/HeaderLink';
+import { LanguageSwitcher } from '@thecointech/site-base/containers/LanguageSwitcher';
+import { CreateAccountButton } from '../../components/AppLinks/CreateAccount';
+import styles from './styles.module.less';
+
+const menuItems = defineMessages({
```

Comment:
We want a single list of headers/pages we can put in both header bar and in burger menu.

Here, I am simply using the key/message combo from defineMessages as route/text.

It's not foolproof, and at some future point I'll be able to update the routes of landing with the compiler-checked versions from App/NFT, unfortunately we do a little bit of complicated routing to for now I just have to make sure we don't mess this up.